README.md

```diff
@@ -106,16 +118,28 @@ git submodule update --init --recursive
 ```
 
 ## Development
+
 See [CONTRIBUTING.md](CONTRIBUTING.md).
 
-## Third party tools
+## Third party tools and integrations
 
-There are also community tools that use OSV. Note that these are community built tools and
-unsupported by the core OSV maintainers.
+There are also community tools that use OSV. Note that these are community built
+tools and unsupported by the core OSV maintainers.
 
-- [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner that uses the OSV database.
+-   [G-Rath/osv-detector](https://github.com/G-Rath/osv-detector): A scanner
+    that uses the OSV database.
+-   [Rust client library](https://github.com/gcmurphy/osv)
+-   [Golang support for the schema](https://pkg.go.dev/golang.org/x/vuln/osv)
+-   [.NET client library and support for the schema](https://github.com/JamieMagee/osv.net)
+-   [EZE-CLI: The one stop shop for security testing in modern development](https://github.com/RiverSafeUK/eze-cli)
+-   [OSS Review Toolkit](https://github.com/oss-review-toolkit/ort)
+-   [Skjold: Security audit python project dependencies against several security
+    advisory databases](https://github.com/twu/skjold)
+-   [Cortex XSOAR](https://github.com/demisto/content)
+-   [dependency track](https://github.com/DependencyTrack/dependency-track)
```

Comment:
nit: Maybe stylise as "Dependency-Track" per the repo description and website?

Reply:
Done. I'd based it off the logo text.